app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Apply again to 3 courses
+description: We changed our policy so that candidates can apply to 3 courses when applying again.
+date: 2022-02-28
+---
+
+Previously, candidates could apply to one course when applying again.
+
+Under the new policy, candidates can apply to 3 courses when applying again.
+
+## Why we changed the policy
+
+The policy where candidates can apply to only one course was inherited from UCAS Teacher Training.
+
+We did not understand why the policy was designed this way.
+
+We wanted to test our hypothesis that having only one course choice disheartened candidates and caused them to drop out of the application process.
+
+Our user research found that:
+
+* candidates expected to have 3 course choices when applying again
+* both candidates and providers felt that the policy penalised candidates for not getting a place
+* candidates felt that having only one course choice signified that they had been considered unsuitable for teaching and that they were being discouraged from applying again
+
+In reality, there are many reasons why someone’s first application does not work out. For example, candidates may not know what to expect at an interview and so do not prepare accordingly.
+
+## Impact on teacher training providers
+
+We felt that changing the policy would not have a significant impact on teacher training providers.
+
+Providers thought that any increased workload could be managed because the number of people who apply again is relatively low. Around 10% of all candidates who submit a first application apply again.
+
+More applications may actually benefit providers, giving them more candidates to choose from.
+
+## What the new policy looks like
+
+![Application task list where candidates are prompted to add 3 courses](prompt-to-add-3-courses.png "Candidates are prompted to add 3 courses from their task list.")
+
+![Course review page where candidates are prompted to add 2 more courses after selecting a course](prompt-to-add-two-more-courses.png "After selecting a course, candidates are prompted to add 2 more courses.")
+
+![Course review page where candidates are prompted to add one more course after selecting 2 courses](prompt-to-add-one-more-course.png "After selecting 2 courses, candidates are prompted to add one more course.")
+
+![Email to candidate when their application is not successful, inviting them to apply again and choose 3 courses.](apply-again-email.png "When an application ends, candidates are invited to apply again and choose 3 courses.")
+
+## Outcome of the policy change so far
+
+On 2 March 2022, 7 working days after launch:
```

Comment:
I can give figures for 9 March 2022:

* 58.3%
* 9%
* 32.7%

Still 2.26 choices.

41 candidates have at least one offer (and many more attending interviews).

Reply:
thanks @MylesJarvis. If I change this, I'll need to change the rest of the section - so are you happy for me to leave it as is? 

I've specified the time frame in the entry, so as long as we're clear on that, i don't think it matters too much 

we can do another entry later with more up to date figures